### DISCUSSION TITLE
Beatmaps, skins, and replays missing after pc reinstall

### DISCUSSION THREAD

**Discussion by PanenkoekOSU** [ppy/osu#28826] [Q&A]
Hello, i recently reinstalled my pc, and when i installed osu! lazer again, and imported the files i saved before, my beatmaps, skins and scores didn't show up. After looking through  a few posts i realized, that i was stupid and only copied the "files" folder in AppData/Roaming/osu , thinking this was all i needed. Is there a way to restore my Beatmaps, or am i gonna have to download all of them again?
Thank you for your help!

**Comment by okolenmi** [comment]
You can't restore them if they were located on system drive. You should to download them again. Try to change songs folder or whole game to another drive to avoid such cases in future.

**Comment by cihe13375** [comment]
If you did not format C: drive during reinstallation, there is a chance that the files in the old installation are available under C:\Windows.old

**Comment by PanenkoekOSU** [comment]
Allright, i will note this down for next time, thank you.
